packages/sdks/output/vue/nuxt2/src/blocks/columns.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <div class="builder-columns div-1083i9ahz0v">
+  <div class="builder-columns div-38o6qlysr38">
     <div
-      class="builder-column div-1083i9ahz0v-2"
+      class="builder-column div-38o6qlysr38-2"
```

Comment:
one random thing we should do at some point is use hashes for this instead of stringified random numbers so these are deterministic (woon't keep showing up in our diffs)

Reply:
this is unrelated to this PR just fYI in case you are seeing these and thinking its janky, it is solvable whenever we have a moment :D

Reply:
Oh yeah, I saw the line of code and just need to get around seeding it with a constant so it stops doing that. It's been driving me nuts 😄

Reply:
https://github.com/BuilderIO/mitosis/blob/a7d1f681d0cb8356054380f6ea47c6b19eac22e5/packages/cli/src/build/build.ts#L38